website/docs/reference/commands/retry.md

```diff
@@ -0,0 +1,22 @@
+---
+title: "About dbt retry command"
+sidebar_label: "retry"
+id: "retry"
+---
+
+`dbt retry` re-executes the last `dbt` command from the node point of failure. If the previously executed `dbt` command was successful, `retry` will finish as a no op. 
```

Comment:
What is a "no op?" If it stands for something we might want to write it out. Or ink to docs about it?

Reply:
I spelled it out since it wasn't apparent to me, either.